RollbarPLCrashReporter/Tests/RollbarPLCrashReporterTests/RollbarPLCrashReporterTests.swift

```diff
@@ -6,7 +6,7 @@ final class RollbarPLCrashReporterTests: XCTestCase {
         // This is an example of a functional test case.
         // Use XCTAssert and related functions to verify your tests produce the correct
         // results.
-        XCTAssertEqual(RollbarPLCrashReporter().text, "Hello, World!")
+        //XCTAssertEqual(RollbarPLCrashReporter().text, "Hello, World!")
```

Comment:
`testExample()` is bare function now.

Reply:
commenting out the empty test as a whole...

Reply:
Empty tests have no value and provide misleading information in reports. It also raises suspicion that the test is broken and hidden and it can result in drop in confidence by our customers.

Reply:
The whole test is commented out at this point.